storage/sqlite3.go

```diff
@@ -122,8 +122,7 @@ func GetBlockRange(ctx context.Context, db *sql.DB, blockOut chan<- []byte, errO
 }
 
 func StoreBlock(conn *sql.DB, height int, hash string, sapling bool, encoded []byte) error {
-	insertBlock := "INSERT INTO blocks (block_height, block_hash, sapling, compact_encoding) values (?, ?, ?, ?)"
-
```

Comment:
Nit: leave the blank line.

Reply:
Pushed a commit myself to address the nit. Now the GitHub change view correctly shows that the only change was `s/INSERT/REPLACE`.